source/extensions/omni.isaac.lab/omni/isaac/lab/managers/reward_manager.py

```diff
@@ -13,28 +13,39 @@
 from typing import TYPE_CHECKING
 
 from .manager_base import ManagerBase, ManagerTermBase
-from .manager_term_cfg import RewardTermCfg
+from .manager_term_cfg import RewardGroupCfg, RewardTermCfg
 
 if TYPE_CHECKING:
     from omni.isaac.lab.envs import ManagerBasedRLEnv
 
 
+DEFAULT_GROUP_NAME = "reward"
```

Comment:
If this is only for internal usage in the class then I suggest marking it with an underscore.